assets/stylesheets/_main.scss

```diff
@@ -54,6 +54,12 @@ body,
 body {
 	background: $gray-light;
 	-ms-overflow-style: scrollbar;
+
+	// This fixes an issue with the `click-outside` package not working on mobile.
+	// Specifically, this makes clicking outside a popover work on touch devices. -shaun
+	@include breakpoint( "<660px" ) {
+		cursor: pointer;
+	}
```

Comment:
@shaunandrews, @lsinger: this displays a wrong cursor when hovering elements that are not clickable on small viewports (I tested this on desktop only). Is that expected?

<img width="205" alt="screenshot" src="https://user-images.githubusercontent.com/594356/37829567-c5a7277c-2e9f-11e8-9536-e6ebca12fd9d.png">

Reply:
Judging from the code: expected, maybe, but definitely not intended. The intention is to make clicking outside of the popover work. 

However, as viewports as small as that will usually not display a cursor anyway (touch screen devices, etc.), I'm tempted to leave this as is. Except if there's a better solution for the problem, of course, or we find that users with pointing devices but super narrow browser windows are a group we should support better.